javascriptv3/example_code/personalize/README.md

```diff
@@ -0,0 +1,81 @@
+- [Create a batch inference job](src/personalize_createBatchInferenceJob.js) (CreateBatchInferenceJobCommand)
+- [Create a batch segment job](src/personalize_createBatchSegmentJob.js) (CreateBatchSegmentJobCommand)
+- [Create a campaign](src/personalize_createCampaign.js) (CreateCampaignCommand)
+- [Create a dataset](src/personalize_createDataset.js) (CreateDatasetCommand)
+- [Create a dataset export job](src/personalize_createDatasetExportJob.js) (CreateDatasetExportJobCommand)
+- [Create a dataset group](src/personalize_createDatasetGroup.js) (CreateDatasetGroupCommand)
+- [Create a dataset import job](src/personalize_createDatasetImportJob.js) (CreateDatasetImportJobCommand)
+- [Create a domain dataset group](src/personalize_createDomainDatasetGroup.js) (CreateDatasetGroupCommand)
+- [Create a domain schema](src/personalize_createDomainSchema.js) (CreateSchemaCommand)
+- [Create an event tracker](src/personalize_createEventTracker.js) (CreateEventTrackerCommand)
+- [Create a filter](src/personalize_createFilter.js) (CreateFilterCommand)
+- [Create a recommender](src/personalize_createRecommender.js) (CreateRecommenderCommand)
+- [Create a schema](src/personalize_createSchema.js) (CreateSchemaCommand)
+- [Create a solution](src/personalize_createSolution.js) (CreateSolutionCommand)
+- [Create a solution version](src/personalize_createSolutionVersion.js) (CreateSolutionVersion)
+- [Get a personalized ranking](src/personalize_getPersonalizedRanking.js) (GetPersonalizedRankingCommand)
+- [Get recommendations (custom dataset group)](src/personalize_getRecommendations.js) (GetRecommendationsCommand)
+- [Get recommendations from a recommender (domain dataset group)](src/personalize_getRecommendationsFromRecommender.js) (GetRecommendationsCommand)
+- [Get recommendation with a filter (custom dataset group)](src/personalize_getRecommendationsWithFilter.js) (GetRecommendationsCommand)
+- [Put events](src/personalize_putEvents.js) (PutEventsCommand)
+- [Put items](src/personalize_putItems.js) (PutItemsCommand)
+- [Put users](src/personalize_putUsers.js) (PutUsersCommand)
+
+**Note**: All code examples are written in ECMAscript 6 (ES6). For guidelines on converting to CommonJS, see 
+[JavaScript ES6/CommonJS syntax](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sdk-examples-javascript-syntax.html).
+
+## Important
+
+- As an AWS best practice, grant this code least privilege, or only the
+  permissions required to perform a task. For more information, see
+  [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the *AWS Identity and Access Management User Guide*.
+- This code has not been tested in all AWS Regions. Some AWS services are
+  available only in specific AWS Regions. For more information, see the
+  [AWS Regional Services List](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/)
+  on the AWS website.
```

Comment:
This code is not tested in every AWS Region. Some AWS services are available only in specific AWS Regions. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/).